### PR TITLE
feat: get cmd for dt and small bug fix

### PR DESCRIPTION
**Pull request by axif0**
### Contributor checklist

- [X] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch
- [X] I have tested my code with the `pytest` command as directed in the [testing section of the contributing guide](https://github.com/scribe-org/Scribe-Data/blob/main/CONTRIBUTING.md#testing)

---

### Description
Added `scribe-data get --all -dt nouns (gets all nouns for all language)` and added more error handling in CLI command.
   

### Related issue

- #478

### CODE REVIEW

**Comment by andrewtavis** [comment]
This is meant to integrate the work from @KesharwaniArpita in #491?

**Comment by axif0** [comment]
> This is meant to integrate the work from @KesharwaniArpita in #491?

Ya.. For avoiding the conflict and simplicity I copied @KesharwaniArpita `s [get.py](https://github.com/scribe-org/Scribe-Data/blob/aa57eedb1fbeb3687379f603a2475e2302fdf655/src/scribe_data/cli/get.py) and then implement further changes.

File: src/scribe_data/cli/get.py [hunk 1]
```diff
@@ -79,44 +79,67 @@ def get_data(
     -------
         The requested data saved locally given file type and location arguments.
     """
-    # MARK: Defaults
+    # Mark: Defaults
```
Comment (andrewtavis) [inline src/scribe_data/cli/get.py:82]: Quick note on marks, @KesharwaniArpita and @axif0: They only work when they're all caps :)

**Review by andrewtavis — APPROVED** [review]
Minor improvements to this, @axif0 :) Thanks for the continued work to improve the user experience! Thanks also @KesharwaniArpita for the initial work here :)

**Pull request merged**